storage/libradb/src/state_store/state_store_test.rs

```diff
@@ -126,7 +126,7 @@ fn test_state_store_reader_writer() {
             (address3, value3.clone()),
         ],
         1, /* version */
-        4, /* expected_nodes_created */
+        6, /* expected_nodes_created */
```

Comment:
I checked with @lightmark to make sure that these changes were expected. Basically: the address format changed and the storage uses the hashes of addresses as keys, so the tree structure this test is expecting also changed.